docs/html-support.md

```diff
@@ -0,0 +1,121 @@
+# Enable HTML Templates in EmailOTP
+
+Use this configurations if you need to use HTML Templates in the Emails.
+**This feature is supported from Identity Server v5.6 onwards**.
+
```

Comment:
Shall we say:

Follow the steps below if you want to use HTML templates in emails when you use the Email OTP authenticator.
**NOTE**: The use of HTML templates is only supported from WSO2 Identity Server version 5.6.0 onwards.